libs/ledger-live-common/src/__tests__/test-helpers/setup.ts

```diff
@@ -1,9 +1,6 @@
 import "./environment";
 import BigNumber from "bignumber.js";
-import { liveConfig } from "../../config/sharedConfig";
-import { LiveConfig } from "@ledgerhq/live-config/LiveConfig";
 
-LiveConfig.setConfig(liveConfig);
```

Comment:
This is used in bridge integration tests right ?

Reply:
yes. We have "import "./environment";" in the same file.
So, it doesn't impact anything.